ext/mbstring/php_mbregex.c

```
@@ -1215,6 +1212,7 @@ PHP_FUNCTION(mb_split)
 
 	/* create regex pattern buffer */
 	if ((re = php_mbregex_compile_pattern(arg_pattern, arg_pattern_len, MBREX(regex_default_options), MBREX(regex_default_syntax))) == NULL) {
+		// TODO throw as invalid regex?
```

Can drop this TODO, we can't throw on invalid regex. (Same for todos below).